src/sidebar/components/mentions/MentionPopoverContent.tsx

```diff
@@ -21,15 +23,25 @@ export default function MentionPopoverContent({
   }
 
   return (
-    <div className="flex flex-col gap-y-1.5">
-      <div data-testid="username" className="text-md font-bold">
-        @{content.username}
+    <div className="flex flex-col gap-y-4">
+      <div className="flex flex-col gap-y-1.5">
+        <div data-testid="username" className="text-md font-bold">
+          @{content.username}
+        </div>
+        {content.display_name && (
+          <div data-testid="display-name" className="text-color-text-light">
+            {content.display_name}
+          </div>
+        )}
       </div>
-      {content.display_name && (
-        <div data-testid="display-name" className="text-color-text-light">
-          {content.display_name}
+      {content.description && (
+        <div data-testid="description" className="line-clamp-2">
```

Comment:
I'm experimenting on cropping the description to up to 2 lines to avoid having an excessively big popover.

On the other hand, descriptions are 250-character long tops, so maybe we can simply let the be shown in full.

The problem with cropping is that perhaps we should then add the full content in the element's title, but for that to work, we need https://github.com/hypothesis/client/issues/6821